crates/sui-protocol-config/src/lib.rs

```diff
@@ -454,6 +524,17 @@ impl ProtocolConfig {
         // To change the values here you must create a new protocol version with the new values!
         match version.0 {
             1 => Self {
+                max_tx_in_batch: Some(10),
+                max_modules_in_publish: Some(8),
+                max_arguments: Some(16),
```

Comment:
64?

Reply:
Increasing this interacts poorly with having multiple Calls in a ProgrammableTransaction per the discussion above (unless I add a bunch of separate limits for ProgrammableTransaction Calls, but even the existing limit for PT you wanted me to increase 🙃)